examples/basic/usage_tracking.py

```diff
@@ -43,4 +43,3 @@ async def main() -> None:
 
 if __name__ == "__main__":
     asyncio.run(main())
-
```

Comment:
You had a formatter make unrelated changes to a bunch of files. They probably won't accept that as-is